jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -1776,6 +1783,7 @@ def dot_product_attention(
       is the index of each token. E.g., if sliding_window_length == 3 and the
       sequence is [0, 1, 2, 3, c, 4, 5], token `c` can attend to [4, 5, c].
     use_fp8: Whether to use FP8 attention mechanism.
+    return_residual: Whether to return softmax stat tensor to users.
```

Comment:
Is "softmax stat tensor" standard terminology? I'm not using attention APIs much myself, so it's possible this is obvious to anyone but me, but I would personally prefer a more clear explanation.

Reply:
I updated the docstring and attach the link to the flash attention paper where it is defined, could you check if that is clear enough for users to understand?